chunky/src/java/se/llbit/chunky/ui/render/AdvancedTab.java

```diff
@@ -226,11 +225,19 @@ public void setController(RenderControlsFxController controls) {
     this.renderControls = controls;
     this.controller = controls.getRenderController();
     scene = controller.getSceneManager().getScene();
-    controller.getRenderer().setOnRenderCompleted((time, sps) -> {
+    controller.getRenderManager().setOnRenderCompleted((time, sps) -> {
       if(shutdownAfterCompletedRender()) {
         // TODO: rewrite the shutdown alert in JavaFX.
         new ShutdownAlert(null);
       }
     });
+
+    if (rendererSelect.getItems().isEmpty()) {
+      RenderManager renderManager = controller.getRenderManager();
+      rendererSelect.getItems().addAll(renderManager.getRenderers());
+      rendererSelect.getSelectionModel().select(renderManager.getRendererName());
```

Comment:
We shouldn't confuse the _name_ and the "short" _id_ of a renderer. The ID should be used to identify the renderer internally, the name should only be used for display (and may change without breaking existing scenes).

The other selection boxes have a similar problem (except for the postprocessing mode), so that's fine for now. Make sure to store the _id_ in the scene description though, not the display name.